src/lxp/inverter.rs

```diff
@@ -55,6 +56,7 @@ impl WaitForReply for Receiver {
                     }
                 }
                 (_, Ok(ChannelData::Packet(_))) => {} // TODO ReadParam and WriteParam
+                (_, Ok(ChannelData::Connected(_))) => {} // Breaks on channel overflow, but we have a timeout
```

Comment:
This isn't a new bug, but introducing the `Connected` message makes it more obvious. Once the inverter is disconnected, the queue might fill up while it waits to connect. If that happens, a message may be silently dropped from the queue, and so never sent when eventually the inverter reconnects. Fortunately, the timeout means this isn't an eternal wait.